testing/runner/Views/Main/RunSuite.cshtml

```diff
@@ -178,7 +178,9 @@
                     'exceljs': '@Url.Content("~/node_modules/exceljs/dist/exceljs.js")',
                     'turndown': '@Url.Content("~/node_modules/turndown/lib/turndown.browser.umd.js")',
                     'showdown': '@Url.Content("~/node_modules/showdown/dist/showdown.js")',
-
+                    'rrule': '@Url.Content("~/node_modules/rrule/dist/es5/rrule.js")',
+                    'luxon': '@Url.Content("~/node_modules/luxon/src/luxon.js")',
```

Comment:
```suggestion
                    'luxon': '@Url.Content("~/node_modules/luxon/build/global/luxon.min.js")',
```

Reply:
@AntonSermyazhko  Does this change fix the IE support problem in luxon?

Reply:
No unfortunately.
We tried to import luxon from dist in order to solve this problem.